primeorder/src/projective.rs

```diff
@@ -332,6 +373,7 @@ where
     Self: Double,
     C: PrimeCurveParams,
 {
+    // TODO optimize impl for r0
```

Comment:
I'm fine with postponing optimizations but it seems odd that this function body is empty

Reply:
ah, I meant to get around to this, this isn't an empty function, it just defaults to the default implementation and there is some overhead going from proj -> affine for the few ops that happen in the default fn. Perhaps reasonable to not override to keep things simple (not showing up on profile meaningfully)